pykafka/pykafka/simpleconsumer.py

```diff
@@ -292,9 +310,8 @@ def _reset_offsets(self, errored_partitions):
             reqs = [owned_partition.build_offset_request(self._auto_offset_reset)
                     for owned_partition in owned_partitions]
             response = broker.request_offset_limits(reqs)
-            for partition_id, offsets in response.topics[self._topic.name].iteritems():
-                owned_partition = self._partitions_by_id[partition_id]
-                owned_partition.set_offset(offsets[0])
+            error_handlers = {0: lambda op, pres: op.set_offset(pres.offset[0])}
```

Comment:
This isn't an error handler, it's a success handler.  I think that's an important distinction and might help clarify `_handle_partition_errors` if instead it's `_handle_partition_responses`.